lib/modules/model/types/ModelContent.ts

```diff
@@ -33,16 +33,27 @@ export interface LocaleDetails {
   description: string;
 }
 
-interface DefinitionMetadataDetails {
+interface BaseDefinition {
   readOnly?: boolean;
-  type?: string;
-  values?: string[] | number[] | boolean[];
+  type: string;
```

Comment:
You can narrow it to `MetadataDetailsEnum`, so it enforces using an enum option when extending `BaseDefinition`.